run.ts

```diff
@@ -24,7 +26,8 @@ async function main() {
             const stderr: string = error.stderr !== undefined ? error.stderr : "";
             let geminiText: [string];
             try {
-                geminiText = await callGemini(stderr, cmd);
+                const lastFiveLines = await lastFiveCommands();
+                geminiText = await callGemini(`my last five commands ran: ${lastFiveLines}, srderr: ${stderr}`, cmd);
```

Comment:
For prompts, I'm guessing a standard 'style' hasn't been established yet, but for me I would define them at the top of the code. They are like constants, after all, and you may want to tweak them without having to dig through the code itself.

Like a `lambda` in Python:

```node
const PROMPT = (a, b, c) => `foo ${a} ${b} ${c}`
```

Reply:
Thanks for the suggestion, I've done something similar with the prompt.
I also relocated the `lastFiveLines` string to the `prompt()` lambda.